cmd/kubelet/app/options/options.go

```diff
@@ -209,6 +212,7 @@ func NewKubeletFlags() *KubeletFlags {
 		MaxContainerCount:                   -1,
 		MaxPerPodContainerCount:             1,
 		MinimumGCAge:                        metav1.Duration{Duration: 0},
+		MaxSurvivalGCAge:                    metav1.Duration{Duration: -1},
```

Comment:
better to keep a week by default?

Reply:
My guess is that it'll be easiest to get this merged if it doesn't change existing behavior.